src/app/code/community/Shopgate/Cloudapi/Block/Customer/Account.php

```diff
@@ -36,6 +36,36 @@ public function getToken()
      */
     public function getSgCloudCallbackData()
     {
-        return $this->getRequest()->getParam('sgcloud_callback_data', '');
+        if ($this->getRequestHelper()->cookieIsSet(Shopgate_Cloudapi_Helper_Request::KEY_SGCLOUD_CALLBACK_DATA)) {
+            return $this->getRequestHelper()->cookieGetValue(
+                Shopgate_Cloudapi_Helper_Request::KEY_SGCLOUD_CALLBACK_DATA
+            );
+        } else {
+            return '';
+        }
+    }
+
+    /**
+     * @return string
+     */
+    public function getSgCloudIsCheckout()
+    {
+        return $this->getRequestHelper()->cookieIsSet(Shopgate_Cloudapi_Helper_Request::KEY_SGCLOUD_IS_CHECKOUT);
+    }
```

Comment:
Please write a wrapper function for that, e. g. isShopgateCheckout, since you are calling the request helper with a constant as parameter that is also defined in the same helper.

Reply:
done